site/content/en/contribute/features/_index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Contributing Features"
+linkTitle: "Contributing Features"
+type: docs
+weight: 30
+description: >
+    How to contribute features
+---
+
+[issue]: https://github.com/kubernetes-sigs/kustomize/issues
+[sig-cli]: https://groups.google.com/g/kubernetes-sig-cli
+[meeting agenda]: https://docs.google.com/document/d/1r0YElcXt6G5mOWxwZiXgGu_X6he3F--wKwg-9UBc29I/edit#heading=h.himo1st0tqyy
+[KEP]: https://github.com/kubernetes/enhancements/tree/master/keps/sig-cli
+[table-driven]: https://github.com/kubernetes-sigs/kustomize/blob/master/api/filters/imagetag/imagetag_test.go#L16
+[eschewed feature list]: /faq/eschewedfeatures/
```

Comment:
this links to another section in the docs. in order not to make this PR too big with other changes, I added the section (so the link is not broken) with some `TODOs` commented